src/Command/Ide/Wizard/IdeWizardCreateSshKeyCommand.php

```diff
@@ -119,9 +119,18 @@ protected function localIdeSshKeyExists(): bool {
    */
   protected function addSshKeyToAgent($filepath, $password): void {
     // We must use a separate script to mimic user input due to the limitations of the `ssh-add` command.
-    $passphrase_prompt_script = __DIR__ . '/passphrase_prompt.sh';
+    // @see https://www.linux.com/topic/networking/manage-ssh-key-file-passphrase/
+    $temp_filepath = $this->localMachineHelper->getFilesystem()->tempnam(sys_get_temp_dir(), 'acli');
+    $this->localMachineHelper->writeFile($temp_filepath, <<<'EOT'
```

Comment:
I'm not too worried about this given that this command is intended only to be run inside of an IDE.